tests/integration/tier0/check-custom-repo/files/rhel7.repo

```diff
@@ -0,0 +1,17 @@
+[rhel-7-server-rpms]
+name=RHEL 7.9 for $basearch
+baseurl=http://rhsm-pulp.corp.redhat.com/content/dist/rhel/server/7/$releasever/$basearch/os/
```

Comment:
Those are internal addresses, right? Just wanted to make sure it is fine to have them here

Reply:
@SpyTec @bocekm is it problematic in some way to have these here?

Reply:
I will tackle this in a separate PR.
Relate issue: https://issues.redhat.com/browse/RHELC-896